resources/js/tasks/components/ParticipantHomeScreen.vue

```diff
@@ -0,0 +1,488 @@
+                                    </span>
+                                    </div>
+                                </b-popover>
+                                <b-button
+                                    v-if="userPermissions.isAdministrator || userPermissions.canEditScreens"
+                                    class="ml-md-2"
+                                    :href="savedsearchDefaultsEditRoute"
+                                >
+                                    <i class="fas fw fa-cog"></i>
+                                </b-button>
+                            </template>
+                            </pmql-input>
+                        </div>
+                        </div>
+                    </div>
+                    <tasks-list
+                        ref="taskList"
+                        :filter="filter"
+                        :pmql="fullPmql"
+                        :columns="columns"
+                        :disable-tooltip="false"
+                        :disable-quick-fill-tooltip="false"
+                        :fetch-on-created="false"
+                        @in-overdue="setInOverdueMessage"
+                        @data-loading="dataLoading"
+                        @tab-count="handleTabCount"
+                        @on-fetch-task="onFetchTask"
+                        :show-recommendations="true"
+                    ></tasks-list>
+                    </div>
+
+                </div>
+                </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    </div>
+  </template>
+  
+  <script>
+ import ListMixin from './ListMixin';
+ import TasksMixin from '../mixins/TasksMixin';
+ import TasksList from './TasksList.vue';
+  
+  export default {
+    name: 'ParticipantHomeScreen',
+    mixins: [TasksMixin, ListMixin],
+    components: {
+      TasksList
+    },
+    props: {
+      taskDraftsEnabled: {
+        type: Boolean,
+        required: true
+      },
+      userFilter: {
+        type: Object,
+        required: true
+      },
+      defaultColumns: {
+        type: Array,
+        required: true
+      },
+      userConfiguration: {
+        type: Object,
+        default: () => ({})
+      },
+      userPermissions: {
+        type: Object,
+        required: true
+      },
+      savedsearchDefaultsEditRoute: {
+        type: String,
+        required: true
+      }
+    },
+    data() {
+      return {
+        showMenu: false,
+        urlConfiguration: "users/configuration",
+        localUserConfiguration: {}
+      }
+    },
+    mounted() {
+      this.defineUserConfiguration();
+      this.$nextTick(() => {
+        if (this.$refs.taskList) {
+          this.$refs.taskList.fetch();
+        }
+      });
+    },
+  }
+  </script>
+  
+  <style lang="scss" scoped>
```

Comment:
**Style duplication**: There are multiple `<style scoped>` blocks that could be combined to improve code readability and maintainability.

Reply:
Resolved